pkgs/by-name/wa/waybar/package.nix

```diff
@@ -163,13 +166,15 @@ stdenv.mkDerivation (finalAttrs: {
       "pulseaudio" = pulseSupport;
       "rfkill" = rfkillSupport;
       "sndio" = sndioSupport;
-      "systemd" = false;
+      "systemd" = true;
```

Comment:
should probably follow the same theme as the other flags if someone wants to build without systemd dependency

Reply:
This option only enabled copying `waybar.service` to the correct location (`PKG_CONFIG_SYSTEMD_SYSTEMDUSERUNITDIR`) during the build, nothing else.

The result waybar binary remains identical as the systemd dependency is never used at runtime, so I don't really see a need to allow disabling it. I even believe a `systemdSupport` option would actually confuse people by making them think the waybar binary can depend on and use the systemd dependency at runtime.

Reply:
That's what `buildInputs` (i.e. `depsHostTarget`) implies, that it's a runtime linkage.
If it's not used at runtime why would it be there?